flow/layers/opacity_layer.cc

```diff
@@ -35,6 +35,10 @@ void OpacityLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
   SkMatrix child_matrix = matrix;
   child_matrix.postTranslate(offset_.fX, offset_.fY);
 
+  // Similar to what's done in TransformLayer::Preroll, we have to apply the
+  // reverse transformation to the cull rect to properly cull child layers.
+  context->cull_rect = context->cull_rect.makeOffset(-offset_.fX, -offset_.fY);
```

Comment:
Do we need to add a mutator to the stack to represent this modification to the context?

Reply:
Hmm, transform_layer doesn't so I guess the answer is probably no...